client/src/main/scala/drt/client/services/JSDateConversions.scala

```diff
@@ -70,6 +70,10 @@ object JSDateConversions {
       override def getTimeZoneOffsetMillis(): MillisSinceEpoch = date.utcOffset().toLong * 60000L
 
       def startOfTheMonth(): SDateLike = SDate(date.getFullYear(), date.getMonth(), date.getDate(), 0, 0)
+
+      def getLocalLastMidnight: SDateLike = SDate(getFullYear(), getMonth(), getDate())
```

Comment:
is this going to always have the correct timezone?

Reply:
Yes, I checked as also wasn't sure. Bu the apply method uses the Europe/London timezone